shared/nm-utils/nm-macros-internal.h

```diff
@@ -866,6 +866,17 @@ fcn (void) \
 #define NM_STR_HAS_PREFIX(str, prefix) \
 	(strncmp ((str), ""prefix"", NM_STRLEN (prefix)) == 0)
 
+#define NM_STR_HAS_SUFFIX(str, prefix) \
+	({ \
+		const char *_str = (str); \
+		gsize _l = strlen (_str); \
+		\
+		(   (_l >= NM_STRLEN (prefix)) \
+		 && (0 == memcmp (&_str[_l - NM_STRLEN (prefix)], \
```

Comment:
Why is 0 on the left side? It seems unusual for our coding style.

Reply:
the reason was, that the `== 0` at the end is easy to miss in
```
»···»···(   (_l >= NM_STRLEN (suffix)) \
»···»··· && (memcmp (&_str[_l - NM_STRLEN (suffix)], \
»···»···             ""suffix"", \
»···»···             NM_STRLEN (suffix)) == 0)); \
```

But I agree. Changed.